picard/util/pipe.py

```diff
@@ -194,7 +194,7 @@ def __generate_filenames(self, app_name: str, app_version: str) -> List[str]:
         for directory in self.PIPE_DIRS:
             if directory:
                 _pipe_names.append(os.path.join(os.path.expanduser(directory),
-                                                f"{app_name}_v{app_version}_{self._identifier}_pipe_file"))
+                                                sanitize_filename(f"{app_name}_v{app_version}_{self._identifier}_pipe_file")))
```

Comment:
You need to import it ;)

Reply:
Already done ;) https://github.com/metabrainz/picard/pull/2139/commits/6c510ed814b77fa9d6e7b4f6387bfef60a0d7ad5